### PR TITLE
[Enhancement] Add install.sh script to automate setup and execution of gravitino-playground

### PR DESCRIPTION
**Pull request by waukin**
## Background
Add install script to automate setup and execution.

## Change
- add install.sh

## Test 
Users can run install.sh in their local environment using the following command:
```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/apache/gravitino-playground/HEAD/install.sh)"
```

## Reference
- https://akarshseggemu.medium.com/bash-script-to-check-docker-installation-0a51f90d27e1
- https://gist.github.com/jwebcat/5122366
- https://stackoverflow.com/questions/16261100/cant-download-github-project-with-curl-command

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (waukin) [reply install.sh:1]: Apache license added in this committ: [d372348a0f64c6d6985f9a02d602db293e8e7a4e](https://github.com/apache/gravitino-playground/pull/92/commits/d372348a0f64c6d6985f9a02d602db293e8e7a4e)

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    echo "Docker is not installed. Please install Docker before proceeding."
+    exit 1
+fi
+
+# Check if Docker Compose is installed
+if ! command -v docker-compose &> /dev/null
+then
+    echo "Docker Compose is not installed. Please install Docker Compose before proceeding."
+    exit 1
+fi
```
[the remark this answers is not in the document]
Comment (waukin) [reply install.sh]: Environment check removed in this commit: [0504c2573f45ae833b4567618dc86856b49a9da3](https://github.com/apache/gravitino-playground/pull/92/commits/0504c2573f45ae833b4567618dc86856b49a9da3)

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Download the source code for gravitino-playground directly from GitHub.
+echo "Downloading gravitino-playground..."
+curl -L -O https://github.com/apache/gravitino-playground/archive/refs/heads/main.zip
+unzip main.zip
```
[the remark this answers is not in the document]
Comment (waukin) [reply install.sh]: I have renamed download file as `gravitino-playground-main.zip`.

**Review by xunliu — COMMENTED** [review]
hi @waukin Thank you for your contributions.
I test in my environment. very good.
```
xun@Xuns-MBP temp % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/xunliu/gravitino-playground/HEAD/install.sh)"
Downloading gravitino-playground...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7279k    0 7279k    0     0   313k      0 --:--:--  0:00:23 --:--:--  807k
Archive:  gravitino-playground-main.zip
26cf298c9f6602c4626823bfb7f036aafc11257f
   creating: gravitino-playground-main/
 extracting: gravitino-playground-main/.gitignore  
  inflating: gravitino-playground-main/LICENSE  
  inflating: gravitino-playground-main/README.md  
  inflating: gravitino-playground-main/docker-compose.yaml  
   creating: gravitino-playground-main/healthcheck/
  inflating: gravitino-playground-main/healthcheck/gravitino-healthcheck.sh  
  inflating: gravitino-playground-main/healthcheck/trino-healthcheck.sh  
   creating: gravitino-playground-main/init/
   creating: gravitino-playground-main/init/common/
  inflating: gravitino-playground-main/init/common/common.sh  
  inflating: gravitino-playground-main/init/common/init_metalake_catalog.sh  
   creating: gravitino-playground-main/init/gravitino/
  inflating: gravitino-playground-main/init/gravitino/gravitino-dependency.sh  
  inflating: gravitino-playground-main/init/gravitino/gravitino.conf  
  inflating: gravitino-playground-main/init/gravitino/init.sh  
   creating: gravitino-playground-main/init/hive/
  inflating: gravitino-playground-main/init/hive/init.sh  
   creating: gravitino-playground-main/init/jupyter/
   creating: gravitino-playground-main/init/jupyter/data/
   creating: gravitino-playground-main/init/jupyter/data/pdfs/
  inflating: gravitino-playground-main/init/jupyter/data/pdfs/Berlin.pdf  
  inflating: gravitino-playground-main/init/jupyter/data/pdfs/Tokyo.pdf  
  inflating: gravitino-playground-main/init/jupyter/data/pdfs/Toronto.pdf  
  inflating: gravitino-playground-main/init/jupyter/gravitino-fileset-example.ipynb  
  inflating: gravitino-playground-main/init/jupyter/gravitino-spark-trino-example.ipynb  
  inflating: gravitino-playground-main/init/jupyter/gravitino-trino-example.ipynb  
  inflating: gravitino-playground-main/init/jupyter/gravitino_llamaIndex_demo.ipynb  
  inflating: gravitino-playground-main/init/jupyter/init.sh  
  inflating: gravitino-playground-main/init/jupyter/jupyter-dependency.sh  
   creating: gravitino-playground-main/init/mysql/
  inflating: gravitino-playground-main/init/mysql/init.sql  
   creating: gravitino-playground-main/init/postgres/
  inflating: gravitino-playground-main/init/postgres/init.sql  
   creating: gravitino-playground-main/init/spark/
  inflating: gravitino-playground-main/init/spark/init.sh  
  inflating: gravitino-playground-main/init/spark/spark-defaults.conf  
  inflating: gravitino-playground-main/init/spark/spark-dependency.sh  
   creating: gravitino-playground-main/init/trino/
  inflating: gravitino-playground-main/init/trino/init.sh  
  inflating: gravitino-playground-main/init/trino/init.sql  
  inflating: gravitino-playground-main/playground.sh  
Would you like to run gravitino-playground immediately? [Y/N]: Y
Starting gravitino-playground...
The playground requires 2 CPU cores, 8 GB of RAM, and 25 GB of disk storage to operate efficiently.
Confirm the requirement is available in your OS [Y/n]:Y
```

**Comment by xunliu** [comment]
hi @waukin Please update gravitino-playground/README.md file

1. https://github.com/apache/gravitino-playground/blob/main/README.md?plain=1#L28
I think git is option

2. https://github.com/apache/gravitino-playground/blob/main/README.md?plain=1#L49
change `Launch playground` to `Use git download and launch playground`

3. https://github.com/apache/gravitino-playground/blob/main/README.md?plain=1#L49
Add a new paragraph to introduce this new future 
maybe `One curl command launch playground`

**Comment by waukin** [comment]
> hi @waukin Please update gravitino-playground/README.md file
> 
> 1. https://github.com/apache/gravitino-playground/blob/main/README.md?plain=1#L28
>    I think git is option
> 2. https://github.com/apache/gravitino-playground/blob/main/README.md?plain=1#L49
>    change `Launch playground` to `Use git download and launch playground`
> 3. https://github.com/apache/gravitino-playground/blob/main/README.md?plain=1#L49
>    Add a new paragraph to introduce this new future
>    maybe `One curl command launch playground`

README.md updated

**Review by xunliu — APPROVED** [review]
@waukin Thank you for your contirbutions.
LGTM

**Pull request merged**